site/src/pages/[shortname]/[blockSlug].page.tsx

```diff
@@ -164,6 +164,30 @@ const BlockPage: NextPage<BlockPageProps> = ({
     return catalog.filter(({ name }) => name !== blockMetadata.name);
   }, [catalog, blockMetadata]);
 
+  const repositoryUrl: URL | null = useMemo(() => {
+    if (typeof blockMetadata.repository === "string") {
+      return new URL(blockMetadata.repository);
+    }
+
+    const { type, url, directory } = blockMetadata.repository ?? {};
+
+    if (url) {
+      let returnUrl = url;
+
+      if (type === "git") {
+        returnUrl = returnUrl.replace(new RegExp(/\.git$/), "");
+
+        if (directory) {
+          returnUrl = `${returnUrl}/tree/main/${directory}`;
```

Comment:
This will return a wrong url if it's not on `main` branch.

Reply:
Should be fixed now.